url-dsl/src/main/scala/urldsl/language/PathSegmentWithQueryParams.scala

```diff
@@ -4,7 +4,7 @@ import app.tulz.tuplez.Composition
 import urldsl.url.{UrlStringDecoder, UrlStringGenerator, UrlStringParserGenerator}
 import urldsl.vocabulary._
 
-final class PathSegmentWithQueryParams[PathType, +PathError, ParamsType, +ParamsError] private[language] (
+final class PathSegmentWithQueryParams[PathType, PathError, ParamsType, ParamsError] private[language] (
     pathSegment: PathSegment[PathType, PathError],
     queryParams: QueryParameters[ParamsType, ParamsError]
```

Comment:
If we're making breaking changes, could you please expose these constructor params as val? Or is there a good reason not to? I feel like this will eventually come in handy for route transformations in Waypoint...